plenum/test/plugin/test_notifier_plugin_manager.py

```diff
@@ -55,11 +55,13 @@ def testPluginManagerSendMessageUponSuspiciousSpikeFailsOnMinCnt(
     newVal = 10
     config = {
         'coefficient': 2,
-        'minCnt': 10
+        'minCnt': 10,
+        'minActivityThreshold': 0,
+        'enabled': True
     }
     assert pluginManagerWithImportedModules\
         .sendMessageUponSuspiciousSpike(topic, historicalData,
-                                        newVal, config, name)is None
+                                        newVal, config, name, True) is None
```

Comment:
Please use named argument for boolean constants

Reply:
done